File: 2588 곱셈/2588.c
#include <stdio.h>
int main() {
	int a, b;
	scanf("%d %d", &a, &b);

	int temp = b;
	while (temp){
		printf("%d\n", a * (temp % 10));
		temp /= 10;
	}
	printf("%d\n", a * b);
	
	return 0;
}

File: 2588 곱셈/2588.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
    static public void main(String[] args) throws IOException {
        Scanner scanner = new Scanner(System.in);

        int a = scanner.nextInt();
        int b = scanner.nextInt();

        int temp = b;
        while (temp != 0) {
            System.out.println(a * (temp % 10));
            temp /= 10;
        }
        System.out.println(a * b);
    }
}
